specs/windows/logical_drives.table

```diff
@@ -0,0 +1,15 @@
+table_name("logical_drives")
+description("Details for logical drives on the system. A logical drive generally represents a single partition.")
+schema([
+    Column("device_id", TEXT, "The drive id, usually the drive name, e.g., 'C:'."),
+    Column("type", TEXT, "The type of disk drive this logical drive represents."),
+    Column("free_space", TEXT, "The amount of free space, in bytes, of the drive."),
```

Comment:
Does this need to be a `TEXT`? Could it be cast to being an `INTEGER`? Is it worth casting it to an `INTEGER` value?